### ISSUE TITLE
0.53: map loading images are too bright

### ISSUE THREAD

**Issue by bmorel** [Unvanquished/Unvanquished#1834]
Screenshot from 0.53:

![2022-06-07-152811_1362x748_scrot](https://user-images.githubusercontent.com/1316300/172393368-44ed46d2-e8ca-4c91-b725-25387d350449.png)

Screenshot from 0.52:

![2022-06-07-152513_1362x748_scrot](https://user-images.githubusercontent.com/1316300/172393420-4d881e71-2c78-4c28-a781-4c607e1658e3.png)

**Comment by slipher** [comment]
This is not a regression but rather a bugfix. Librocket did some goofy thing where it would multiply the color of an image with the font color or something. @DolceTriade often complained about this. You can especially clearly see it by writing a chat message combining colors with certain emoticons.

If you open `meta/plat23/plat23.jpg` from the dpk you can see that the 0.53 look is the correct one.

**Comment by illwieckz** [comment]
Yep, it is a bug fix. In fact that's what motivated the move to RmlUi in the first place and made it an urge: there was no correct way to display images properly without putting workarounds every where (workaround we didn't wrote).

See:

- https://github.com/Unvanquished/libRocket/issues/7
- https://github.com/Unvanquished/Unvanquished/issues/1152

On a side note, I would like to get the map name and author being read from `.arena` file or something like that, so we have full control on the font, display ratio, etc. in game, so we would be able to apply filters at display time if people complains it is too bright without having to edit the image themselves.

**Comment by bmorel** [comment]
> so we would be able to apply filters at display time if people complains it is too brigh

Yes, please.

**Comment by illwieckz** [comment]
We can probably already do that by CSS (but relying on features instead of bugs), with the drawback that it will also apply the filter to the text.

On my end I prefer when the level shot is displayed with same colors than the real game instead of being faded out but that's just my opinion and taste.

**Comment by bmorel** [comment]
The problem is that some of those levelshots can really hurt, especially when there's low light environment (night).
If that was not intentional and a bug, it's good that it's fixed, but this makes this ticket a feature request then :)

**Comment by necessarily-equal** [comment]
For the record, this is probably quite easy to do. The "usual html" (like browser html) way to do that would be to set the image opacity to 0.7 and a black background. (Another less elegant way could be to put a #000000c0 layer above the image). In any case, when there is no levelshot the screen displayed is now pure white and this is terrible.

As to whether this should be done. Please yes. I don't want to be woken up on the loading screen where there is nothing to be done. We need more chill moments in Unvanquished.

**Comment by illwieckz** [comment]
The solution involving black background with image transparency would probably also fix the problem of the pure white background destroying the eyes when level shot is missing.
